lib/nexmo_developer/app/views/static/landing_page_documentation.html.erb

```diff
@@ -2,14 +2,26 @@
   <div class="center spacious content-block">
     <h1><%= @landing_info['title'].html_safe %></h1>
     <p><%= raw @landing_info['subtitle'].html_safe %></p>
-  </div>
+    <p>
+      <!-- This is the buttons group area-->
+      <strong>Jump to: </strong>
+      <% @landing_info['top-section'].each do |product| %>
+        <a href="#<%=product.keys.first.to_s.gsub(' ', '-')%>"  class='Vlt-btn Vlt-btn--blue Vlt-btn--app Vlt-btn--small'><%= product.keys.first %></a>
```

Comment:
What do you think about using `product.keys.first.parameterize` to create IDs instead of manually doing `product.keys.first.to_s.gsub(' ', '-')`?